bosh-setup/README.md

```diff
@@ -4,11 +4,13 @@
     <img src="http://azuredeploy.net/deploybutton.png"/>
```

Comment:
2nd sounds good. Doesn't look like qingfuwang would be maintaining this anymore.